components/ExitBetaModal.js

```diff
@@ -16,7 +18,7 @@ export default function ExitBetaModal(props) {
           role="heading"
           aria-level="1"
         >
-          {props.popupTitle}
+          {sentenceCase(props.popupTitle)}
```

Comment:
I'm not sure this is necessary. Given we are in control of our own content here, we can just change the casing in AEM

Reply:
I agree. This looks to be a change that content would make and not a dev change.

Reply:
Fixed.